packages/angular_devkit/architect/node/node-modules-architect-host.ts

```diff
@@ -203,7 +203,21 @@ export class WorkspaceNodeModulesArchitectHost implements ArchitectHost<NodeModu
       );
     }
 
-    const schemaText = readFileSync(path.join(buildersManifestDirectory, schemaPath), 'utf-8');
+    // The file could be either CommonJS or in the local manifest directory.
+    // CommonJS is tried first then reading the file from the manifest directory if loading fails.
+    try {
+      schemaPath = localRequire.resolve(schemaPath, {
+        paths: [basePath],
```

Comment:
```suggestion
        paths: [buildersManifestDirectory],
```
The schema path should be resolved from the manifest directory to prevent unintentionally using a matching file from the base path.  This will also ensure that package reference schema paths can find direct dependencies of the containing builder package.

Reply:
Very good, I like it